src/java/org/apache/cassandra/index/sai/disk/v1/sortedterms/SortedTermsWriter.java

```diff
@@ -49,9 +44,9 @@
  * <p>
  * For documentation of the underlying on-disk data structures, see the package documentation.
  * <p>
- * The TERMS_DICT_ constants allow for quickly determining the id of the current block based on a point id
- * or to check if we are exactly at the beginning of the block.
- * Terms data are organized in blocks of (2 ^ {@link #TERMS_DICT_BLOCK_SHIFT}) terms.
+ * The {@code cassandra.sai.sorted_terms_block_shift} property is used to quickly determine the id of the current block
```

Comment:
Nit: This can be a reference to `{@link CassandraRelevantProperties#SAI_SORTED_TERMS_BLOCK_SHIFT}`

Reply:
This now changes to `{@link #blockShift}` because the value is now passed to the constructor.